src/Psalm/Codebase.php

```diff
@@ -1229,8 +1230,28 @@ public function getCompletionItemsForClassishThing(string $type_string, string $
                             null,
                             (string)$method_storage->visibility,
                             $method_storage->cased_name,
-                            $method_storage->cased_name . '()'
+                            $method_storage->cased_name . '(' . implode(
+                                ', ',
+                                array_map(function (FunctionLikeParameter $parameter) {
+                                    /** @var int $i */
+                                    static $i = 0;
+                                    ++$i;
+                                    return '${'
+                                        . $i
+                                        . ':'
+                                        . (
+                                            $parameter->type !== null
```

Comment:
Argument 3 of str_replace expects array<array-key, mixed>|string, Psalm\Type\Union provided with a __toString method
```
                                                ? str_replace(['$', '}'], ['\$', '\}'], $parameter->type) . ' '
                                                                                        ^^^^^^^^^^^^^^^^
```